### ISSUE TITLE
Added fragment sometimes overlays the snackbar

### ISSUE THREAD

**Issue by MichaelFlisar** [nispok/snackbar#27]
I see following problem:

When I first add a fragment and this fragment shows a snackbar, everything works. If I remove the fragment and readd a new one, the snackbar shown from the fragment is behind this new fragment... I see it, because when I press back and remove the new fragment and reattach the previous one again, I see the snackbar over the previous fragment...

What works for me, as a work around, is following:
```
mCurrentSnackbar.show(parent);
mCurrentSnackbar.postDelayed(new Runnable()
{
    @Override
    public void run()
    {
        mCurrentSnackbar.bringToFront();
    }
}, 500);
```

**Comment by h6ah4i** [comment]
Hi. I am also facing this issue. I have fixed the issue by adding `SnackbarManager.show(Snackbar, ViewGroup)` method.

Forked code: https://github.com/h6ah4i/snackbar/tree/feature/attach_to_non_activity_root_view

```java
ViewGroup snackbarContainer = (ViewGroup) findViewById(R.id.snackbar_conatiner);
SnackbarManager.show(new Snackbar(), snackbarContainer);
```

```xml
<RelativeView>
    <FrameLayout android:id="fragment_container" />
    <FrameLayout android:id="snackbar_container" android:layout_alignParentBottom="true" />
</RelativeView>
```

Related to #76, #62.

**Comment by wmora** [comment]
change included in [2.9.0](https://github.com/nispok/snackbar/releases/tag/v2.9.0)
